guides/installing/docker.md

```diff
@@ -0,0 +1,123 @@
+```yml
+login:
+        image: "rathena:local"
+        container_name: "rathena-login"
+        command: sh -c "/bin/wait-for db:3306 -- /rathena/login-server"
+        ports:
+            - "6900:6900" # login server
+        volumes:
+            - "../..:/rathena" # mount git repo directory inside container
+            - "./asset/inter_conf.txt:/rathena/conf/import/inter_conf.txt" # load db connection
+            - "./asset/char_conf.txt:/rathena/conf/import/char_conf.txt"   #localdev login-char relation
+            - "./asset/map_conf.txt:/rathena/conf/import/map_conf.txt"     #localdev char-map relation
+        init: true # helps with signal forwarding and process reaping
+        tty: true
+        stdin_open: true
+        build:
+            context: .
+            dockerfile: Dockerfile
+        depends_on:
+            - db
+```
+
+### Setting up the server
+1. After cloning the rAthena repo, you should open the project folder in the terminal. In my case I've cloned the project to `Documents/Personal/rathena`, so I've cd'ed to that folder.
+
+<img width="575" alt="console" src="https://user-images.githubusercontent.com/13068064/229590462-9e7a85eb-6e98-4dfb-99ca-4d647ee0b819.png">
+
+2. Then we can cd into `tools/docker` and then we can run the best command in the entire world `docker-compose up`. The very first time we do that command, it will pull the images from Docker hub and then build the server with the parameters specified in the `builder` service.
+<img width="1506" alt="docker-compose up" src="https://user-images.githubusercontent.com/13068064/229595301-777c7e92-08f7-4a02-9c60-ca1ad440001a.png">
+
+3. If we observe the logs, we'll be able to see that the `rathena-builder` container is yielding a bunch of logs. If you ever compiled with `make` you'll find those familiar.
+<img width="545" alt="builder logs" src="https://user-images.githubusercontent.com/13068064/229595637-946c2541-d04c-4f65-87b1-2e48ca2659e5.png">
+ 
+ 4. After a while we can see that the compilation has finished once we see `rathena-builder exited with code 0`
+<img width="616" alt="build finished" src="https://user-images.githubusercontent.com/13068064/229597246-9b00646b-e13d-4c4f-8212-8222023ab605.png">
+
+5. Now all we gotta do is to `ctrl+c` to stop everything and then `docker-compose up`
+<img width="1356" alt="start" src="https://user-images.githubusercontent.com/13068064/229597566-a8d38ab9-f2a6-4848-a95c-1fb5af1d39af.png">
+
+And that's all about it. You've installed one thing on your host computer (a bit more if you count the depdencies to get docker runner) and now you have rAthena running. Once you stop the containers it will be like you never had a rAthena running on your machine.
+
+### Recompiling
+If you want to recompile so your changes are applied, all you gotta do is
```

Comment:
You're obsessed with the phrase "gotta"!

Reply:
All you gotta do is accept that